### PR TITLE
Add scheduler tuning guide.

### PR DESCRIPTION
**Pull request by kfyharukz**

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -176,6 +176,20 @@ This way, `topolvm-scheduler` is exposed by LoadBalancer service.
 
 Then edit `urlPrefix` in [./scheduler-config/scheduler-policy.cfg](./scheduler-config/scheduler-policy.cfg) to specify the LoadBalancer address.
 
+### Scoring tuning
+
+The prioritize of Pod scheduling by `topolvm-scheduler` can be tuned via the following two ways:
+1. Adjust the divisor parameter in the scoring expression
+2. Change the weight for the scoring by `topolvm-scheduler` against the default by kube-scheduler
```
Comment (satoru-takeuchi) [inline deploy/README.md]: "scoring by `topolvm-scheduler`" seems verbose. I consider "Weight  for the scoring" is enough. This indication applies to the same expressions in the following descriptions.

File: deploy/README.md [hunk 2]
```diff
@@ -176,6 +176,20 @@ This way, `topolvm-scheduler` is exposed by LoadBalancer service.
 
 Then edit `urlPrefix` in [./scheduler-config/scheduler-policy.cfg](./scheduler-config/scheduler-policy.cfg) to specify the LoadBalancer address.
 
+### Scoring tuning
+
+The prioritize of Pod scheduling by `topolvm-scheduler` can be tuned via the following two ways:
```
Comment (satoru-takeuchi) [inline deploy/README.md]: `s/prioritize/priority/` is better since `prioritize` is verb.

File: deploy/README.md [hunk 3]
```diff
@@ -176,6 +176,20 @@ This way, `topolvm-scheduler` is exposed by LoadBalancer service.
 
 Then edit `urlPrefix` in [./scheduler-config/scheduler-policy.cfg](./scheduler-config/scheduler-policy.cfg) to specify the LoadBalancer address.
 
+### Scoring tuning
```
Comment (ymmt2005) [inline deploy/README.md]: I think the object of scoring should be specified.
```suggestion
### How to tune the node scoring
```

File: deploy/README.md (hunk 2 above)
Comment (ymmt2005) [inline deploy/README.md]:
```suggestion
The node scoring for Pod scheduling can be fine-tuned with the following two ways:
```

File: deploy/README.md [hunk 4]
```diff
@@ -176,6 +176,20 @@ This way, `topolvm-scheduler` is exposed by LoadBalancer service.
...
+
+The prioritize of Pod scheduling by `topolvm-scheduler` can be tuned via the following two ways:
+1. Adjust the divisor parameter in the scoring expression
```
Comment (ymmt2005) [inline deploy/README.md]:
```suggestion
1. Adjust `divisor` parameter of `topolvm-scheduler`
```

File: deploy/README.md (hunk 1 above)
Comment (kfyharukz) [reply deploy/README.md]: I think so about the first one, but about the latter one,
>So if you want to give a priority to the scoring by `topolvm-scheduler`,

I think `by topolvm-scheduler` seems to be needed to specify the object.

**Pull request merged**